src/widgets/company/InfoConditions/ui.tsx

```diff
@@ -1,3 +1,8 @@
+import { useContext } from 'react';
+
+import { ModalContext } from '@/entities/Modal';
+
+import { modalWindows } from '@/shared/modal/modalWindows';
 import { StFlex } from '@/shared/styles/global';
 
 import InfoConditionsLine1 from './assets/infoConditionsLine1.svg';
```

Comment:
```suggestion
import { useContext } from 'react';

import { modalWindows } from '@/shared/modal/modalWindows';
import { StFlex } from '@/shared/styles/global';

import { ModalContext } from '@/entities/Modal';

import InfoConditionsLine1 from './assets/infoConditionsLine1.svg';
```
Скорее всего eslint подчеркивал, что неверный порядок импортов.